src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect } from 'react';
+import { getPeople } from '../../api';
+import { usePeoplePageContext }
+  from '../../pages/PeoplePage/PeoplePageContext/PeoplePageContext';
+import { Loader } from '../Loader';
+import { Person } from '../Person';
+import { PeopleLoadingError } from '../PeopleLoadingError';
+import { NoPeopleOnServer } from '../NoPeopleOnServer';
+
+export const PeopleTable: React.FC = () => {
+  const {
+    people,
+    loading,
```

Comment:
boolean variables should include is or has!

Reply:
> boolean variables should include is or has!

Good point